docs: document Store search ranking and Console personalization (#2653)
apify/apify-docs: sources/academy/build-and-publish/apify-store-basics/how_store_works.md

Learn more about how to handle the [Issues tab](/academy/actor-marketing-playbook/interact-with-users/issues-tab)

## How Apify Store search works

Search ranking evaluates parameters similar to those in the [Actor quality score](/platform/actors/publishing/quality-score). As a result, the two correlate strongly: Actors with higher quality scores tend to rank higher in [Apify Store](https://apify.com/store) search and the [Apify MCP server](/platform/integrations/mcp) `search-actors` tool, though no specific position is guaranteed.

> To improve your Actor's visibility in search, focus on improving its quality score. See [Actor quality score](/platform/actors/publishing/quality-score) for what the score measures and how to influence each category.

Reviewer: ```suggestion
To improve your Actor's visibility in search, focus on improving its quality score. For what the score measures and how to influence each category, see [Actor quality score](/platform/actors/publishing/quality-score).
```
Starting with the goal would be better

### Search in Apify Console is personalized

> Search in [Apify Console](https://console.apify.com) is an exception: results are personalized for each individual user, so the same query can return different Actors than the public Store at [apify.com/store](https://apify.com/store). Quality score still correlates with ranking - the order is just adjusted per user.

Reviewer: ```suggestion
In [Apify Console](https://console.apify.com), search results are personalized for each user. As a result, the same query can return different Actors than in [Apify Store](https://apify.com/store). Quality score still correlates with ranking, but the order is adjusted per user.
```
To be an exception, there needs to be a rule, and we're talking only about two types here. I'd also stick with linked `Apify Store` instead of introducing URLs. This way, it plays off the previous section better.
